codalab/apps/web/templates/web/competitions/view.html

```diff
@@ -229,6 +229,29 @@ <h3 class="panel-title"><strong>Competition Ends</strong></h3>
             {% endfor %}
         </div>
     </section>
+
+    {% if topthreeleaders %}
+        <div class="phase-container" style="border: 1px solid grey;">
```

Comment:
Try out the same `panel` and `panel-body` classes as on the queues page -- I think that may fit nicer with the top? 

Also, maybe we can make the top three leaderboard take up 1/3rd of the space and 2/3rds on the bottom could be the graph?